packages/react-liveness/src/components/FaceLivenessDetector/shared/Toast.tsx

```diff
@@ -1,28 +1,42 @@
 import * as React from 'react';
-import { Flex, View, ViewProps } from '@aws-amplify/ui-react';
+import { Flex, View, ViewProps, useTheme } from '@aws-amplify/ui-react';
 
 import { LivenessClassNames } from '../types/classNames';
 
 interface ToastProps extends ViewProps {
   variation?: 'default' | 'primary' | 'error';
   size?: 'medium' | 'large';
   heading?: string;
+  isInitial?: boolean;
 }
 
 export const Toast: React.FC<ToastProps> = ({
   variation = 'default',
   size = 'medium',
   children,
+  isInitial = false,
   ...rest
 }) => {
+  const { tokens } = useTheme();
   return (
     <View
       className={`${LivenessClassNames.Toast} ${LivenessClassNames.Toast}--${variation} ${LivenessClassNames.Toast}--${size}`}
-      maxWidth={{ base: '100%', small: '70%' }}
+      maxWidth={{ base: '100%' }}
+      {...((variation === 'primary' || variation === 'error') && {
```

Comment:
nit: could probably declare this styling object outside the function so a new object isn't generated each time.